server/.gitignore

```diff
@@ -0,0 +1,3 @@
+/world
```

Comment:
If you want to ignore stuff that's only relevant for you, please put it in .git/info/exclude rather than .gitignore so that it isn't committed.